dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -1339,4 +1340,10 @@ private void validateVersusMinimumDate(FacesContext context, UIComponent toValid
             context.addMessage(toValidate.getClientId(context), message);
         }
     }
+    
+    public void datasetUnlockedInfo() {
```

Comment:
Maybe a javadoc describing context? Like this is made because of WCG. Otherwise I would had no idea why it was even created, since from non-WCG perspective it is weird why won't you just refresh.